src/bindgen/library.rs

```diff
@@ -873,6 +914,16 @@ impl GeneratedBindings {
         }
 
         if self.config.language == Language::Cxx {
+            out.new_line();
```

Comment:
To separate those implementations visually from the functions before.

One new line is needed to start a new line after a function definition, the second is inserted to have an empty line between functions and classes.